sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5079,7 +5055,7 @@ pi_result _pi_kernel::initialize() {
 
   // Set up how to obtain kernel properties when needed.
   ZeKernelProperties.Compute = [this](ze_kernel_properties_t &Properties) {
-    ZE_CALL_NOCHECK(zeKernelGetProperties, (ZeKernel, &Properties));
+    ZE_CALL(zeKernelGetProperties, (ZeKernel, &Properties));
```

Comment:
Getting a warning here:
```
sycl/plugins/level_zero/pi_level_zero.cpp: In lambda function:
sycl/plugins/level_zero/pi_level_zero.cpp:5059:3: warning: control reaches end of non-void function [-Wreturn-type]
 5059 |   };
```

Reply:
Fixed! Another small test change that slipped in.